authBackEnd/apps/dashboard/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class DashboardConfig(AppConfig):
+    name = 'apps.dashboard'
```

Comment:
can you rename it to something other than apps.dashboard? I feel like something like auth.dashboard would work better

Reply:
the apps in the apps.dashboard is the apps folder inside the authBackEnd folder, I think it will not make any difference to change the name.